### PR TITLE
Remove mint multiple from Vault.

### PR DESCRIPTION
**Pull request by DanielVF**
We are not longer user mintMultiple from the official OUSD DApp, and no longer have a need for it in the contract. Removing to reduce contract complexity.

----

Contract change checklist:
  - [ ] Code reviewed by 2 reviewers. See [template](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review.md) and [example](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review-Example.md).
  - [x] Unit tests pass
  - [x] Slither tests pass with no warning
  - [x] Echidna tests pass if PR includes changes to OUSD contract (not automated, run manually on local)

### CODE REVIEW

**Review by franckc — APPROVED** [review]
LGTM!

File: contracts/contracts/vault/VaultCore.sol [hunk 1]
```diff
@@ -96,69 +96,7 @@ contract VaultCore is VaultStorage {
...
-        uint256[] calldata _amounts,
-        uint256 _minimumOusdAmount
-    ) external whenNotCapitalPaused nonReentrant {
-        require(_assets.length == _amounts.length, "Parameter length mismatch");
-
-        uint256 unitAdjustedTotal = 0;
-        uint256 priceAdjustedTotal = 0;
-        uint256[] memory assetPrices = _getAssetPrices();
-        for (uint256 j = 0; j < _assets.length; j++) {
-            // In memoriam
```
Comment (tomlinton) [inline contracts/contracts/vault/VaultCore.sol:117]: Where does this live now? 😢

File: contracts/contracts/vault/VaultCore.sol [hunk 2]
```diff
@@ -96,69 +96,7 @@ contract VaultCore is VaultStorage {
...
-        for (uint256 i = 0; i < _assets.length; i++) {
-            IERC20 asset = IERC20(_assets[i]);
-            asset.safeTransferFrom(msg.sender, address(this), _amounts[i]);
-        }
-
-        if (unitAdjustedTotal >= autoAllocateThreshold) {
-            _allocate();
-        }
-    }
+    // In memoriam
```
Comment (tomlinton) [inline contracts/contracts/vault/VaultCore.sol:99]: Should've scrolled down!

**Review by tomlinton — APPROVED** [review]
Less code ❤️

**Pull request merged**